app/Http/Controllers/Backend/UserController.php

```diff
@@ -69,4 +70,27 @@ public function show($id)
         $user = $this->userService->showUsers($id);
         return view('admin.users.show', ['user' => $user]);
     }
+
+     /**
+     * Edit for the user.
+     *
+     * @param id
```

Comment:
Cách mô tả `param` như sau:
`@param [type data] [variable] [description for variable]`
Ví dụ: 
`@param int $id User id`